tfgrid3/ubuntu22.04/microvm/README.md

```diff
@@ -0,0 +1,17 @@
+# Development Guide for Ubuntu:22.04 image
+## This guide will take you through steps for pulling & running ubuntu:22.04 image
+
+### Pull the image
+```bash
+docker pull threefolddev/ubuntu:22.04
```

Comment:
images are not pushed there on threefolddev docker hub account 
![Screenshot from 2024-05-16 11-52-35](https://github.com/threefoldtech/tf-images/assets/26298009/eca020f6-45cd-4fe7-9747-6881233bdc2a)

- so it's needed to push latest image there to keep this doc valid .

Reply:
pushed docker images